^ Those changes were needed so the tests could run without error. Now test case 3 fails with:
```
java.lang.AssertionError: Different sets of key nodes are forward reachable from:
Address: 3609c9
Attributes: {
    XCSG.ModelElement.id: "[[/home/tdeering/git/FlowMiner/edu.iastate.flowminer.test/test_resources/testng-6.8.21.jar#javax.xml.parsers.SAXParserFactory org.testng.xml.XMLParser.loadSAXParserFactory()], analyzer=dataflow###return]"
    XCSG.ModelElement.name: "return"
}
Tags: {
    XCSG.Language.Java
    XCSG.ModelElement
    XCSG.Language.Jimple
    XCSG.Node
    jimple.touched
    XCSG.ReturnValue
    XCSG.DataFlow (Node)
    ##index
}
	at org.junit.Assert.fail(Assert.java:88)
	at edu.iastate.flowminer.test.SoundnessCompletenessTest.testEquivalentFlows(SoundnessCompletenessTest.java:127)
```
![junit-4 12 jar_1487521682409](https://cloud.githubusercontent.com/assets/537361/23104238/5e54845a-f68f-11e6-9bce-dc6733edd541.png)

And test case 4 fails with:
```
java.lang.AssertionError: Different sets of key nodes are forward reachable from:
Address: 417711
Attributes: {
    XCSG.ModelElement.id: "[[/home/tdeering/git/FlowMiner/edu.iastate.flowminer.test/test_resources/junit-4.12.jar#void junit.runner.BaseTestRunner.setPreferences(java.util.Properties)], analyzer=v0]"
    XCSG.ModelElement.name: "parameter0"
    XCSG.Parameter.parameterIndex: "0"
}
Tags: {
    XCSG.CallInput
    XCSG.Language.Java
    XCSG.Language.Jimple
    XCSG.Parameter
    XCSG.ModelElement
    XCSG.Node
    jimple.touched
    XCSG.Variable
    ##index
}
	at org.junit.Assert.fail(Assert.java:88)
	at edu.iastate.flowminer.test.SoundnessCompletenessTest.testEquivalentFlows(SoundnessCompletenessTest.java:127)
...
```
![testng-6 8 21 jar_1487521658847](https://cloud.githubusercontent.com/assets/537361/23104236/4b3ef92c-f68f-11e6-9d64-1fa3a173686f.png)

--- a/edu.iastate.flowminer.test/src/edu/iastate/flowminer/test/SoundnessCompletenessTest.java
+++ b/edu.iastate.flowminer.test/src/edu/iastate/flowminer/test/SoundnessCompletenessTest.java
@@ -122,8 +122,9 @@ public class SoundnessCompletenessTest {
 				problem = problem.union(u.edgesTaggedWithAny(XCSG.Contains).reverse(problem));
 				
 				
+				File out = new File(System.getProperty("user.home") + File.separator + toSummarizeFilename + "_" + System.currentTimeMillis() + ".svg");
 				IMarkup markup = new MarkupFromH(h);
-				Job j = SaveUtil.saveGraph(new File("/home/tdeering/" + toSummarizeFilename + "_" + System.currentTimeMillis() + ".svg"), problem.eval(), markup);
+				Job j = SaveUtil.saveGraph(out, problem.eval(), markup);
 				j.join();
 				
 				fail("Different sets of key nodes are forward reachable from:\n" + keyNode);

--- a/edu.iastate.flowminer/.classpath
+++ b/edu.iastate.flowminer/.classpath
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="output" path="bin"/>

--- a/edu.iastate.flowminer/.settings/org.eclipse.jdt.core.prefs
+++ b/edu.iastate.flowminer/.settings/org.eclipse.jdt.core.prefs
@@ -1,7 +1,7 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
-org.eclipse.jdt.core.compiler.compliance=1.7
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.source=1.7
+org.eclipse.jdt.core.compiler.source=1.8

--- a/edu.iastate.flowminer/META-INF/MANIFEST.MF
+++ b/edu.iastate.flowminer/META-INF/MANIFEST.MF
@@ -18,7 +18,7 @@ Require-Bundle: org.eclipse.ui,
  com.opencsv;bundle-version="1.0.0",
  org.junit,
  com.ensoftcorp.atlas.ui;bundle-version="2.1.9"
-Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Export-Package: edu.iastate.flowminer,
  edu.iastate.flowminer.actions,

--- a/edu.iastate.flowminer/src/edu/iastate/flowminer/actions/MineAndExport.java
+++ b/edu.iastate.flowminer/src/edu/iastate/flowminer/actions/MineAndExport.java
@@ -139,7 +139,7 @@ public class MineAndExport extends FlowMinerAction {
 					sm.setTaskName(msg);
 					Collection<IMappingSettings> indexingSettings = new ArrayList<IMappingSettings>(1);
 					indexingSettings.add(new JavaIndexingSettings(OptionIndexJars.INDEX_RELEVANT));
-					IndexingUtil.indexWithSettings(true, indexingSettings, project);
+					IndexingUtil.indexWithSettings(false, indexingSettings, project);
 					sm.worked(23828);
 					
 					/*

